.github/scripts/02-install-pimcore.sh

```diff
@@ -71,7 +70,11 @@ docker compose exec -T php bin/console pimcore:bundle:install PimcoreCopilotBund
 
 cp ../../platform-version/.github/files/config-config.yaml ./config/local
 
+#docker compose exec -T php bin/console doctrine:schema:update --force
+docker compose exec -T php bin/console cache:clear
+docker compose exec -T php bin/console doctrine:migrations:migrate --no-interaction
```

Comment:
normally there should not be any migrations necessary after a fresh installation, right?